sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -4,11 +4,11 @@
 
 ```ts
 
+import { AzureKeyCredential } from '@azure/core-auth';
+import { KeyCredential } from '@azure/core-auth';
 import { OperationOptions } from '@azure/core-http';
 import { PipelineOptions } from '@azure/core-http';
-import { ServiceClientCredentials } from '@azure/core-http';
-import { TokenCredential } from '@azure/identity';
-import { WebResource } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-auth';
```

Comment:
not part of the PR so out of curiosity, this type used to be in the identity package. Did it moved to `core-http`?

Reply:
talked offline